plato/draw/pythreejs/Scene.py

```diff
@@ -69,17 +49,24 @@ def zoom(self, value):
     @property
     def rotation(self):
         camera = self._backend_objects['camera']
-        return self._get_camera_quat(camera)
+        norm_out = -np.array(camera.position)
+        norm_out /= np.linalg.norm(norm_out)
+        norm_up = np.array(camera.up)
```

Comment:
I don't remember all the details of which attributes were actually getting transmitted properly and used by pythreejs with respect to the camera, but the old version of this method only uses `camera.position`, not anything else from the camera. I'd say that's probably why the method isn't working as expected now; did you try with the old version?